autojump.py

```diff
@@ -1,6 +1,8 @@
+
 import ranger.api
 import subprocess
 from ranger.api.commands import *
+import os
```

Comment:
Is that import necessary?

Reply:
Good catch — I was using it for os.path temporarily, but ultimately it was not needed.